alpha/io.h

```diff
@@ -0,0 +1,32 @@
+#include <stdint.h>
+#include <FunctionalInterrupt.h>
+#include <Arduino.h>
+#include "hal.h"
+
+#ifndef __TS_IO__
+#define __TS_IO__
+
+enum TS_ButtonState
+{
+  NotPressed,
+  Short,
+  Long,
+};
+
+class IOButton
```

Comment:
can we prefix with TS_ ?
helps to reduce namespace collision with other libraries

Reply:
iobutton.h
class TS_IOButton